js/plugin/connection.ts

```diff
@@ -27,7 +28,30 @@ export const ConnectionMixin = (SuperClass) => {
       this.dispatchEvent(new CustomEvent(event, { detail, bubbles: true }));
     }
 
+    private onConnectionReady = () => {
```

Comment:
I find calling this `onConnectionReady` as really it is handling the component connection, which I think can be simple `onConnection`

Reply:
Note other comment now on 'ready' versus 'connected'.